.travis.yml

```diff
@@ -0,0 +1,5 @@
+sudo: false
+language: ruby
+rvm:
+- 2.3.1
+before_install: gem install bundler -v 1.13.6
```

Comment:
#nit I don't think this is actually needed for travis. At least I don't remember having to include this line.